pkg/testutils/sensors/load.go

```diff
@@ -15,6 +16,7 @@ type SensorProg struct {
 	Name  string
 	Type  ebpf.ProgramType
 	NotIn bool
+	RegEx bool
```

Comment:
> At the moment regular expression is implemented via strings.Contains
> call, because it's enough for now.

I think calling it regex is confusing. 

How about have an enum?

```go
type ProgMatch = int
const (
      ProgMatchFull ProgMatch = iota // ==
      ProgMatchPartial // strings.Contains()
)
```

we can add a regex later if needed.

Reply:
yep, sounds good, thanks